content/pages/docs/faq/index.mdx

```diff
@@ -1,130 +0,0 @@
----
-title: Frequently Asked Questions
-excerpt: Frequently asked questions about KittyCAD and the Zoo Modeling App (ZMA).
----
-
-### General
```

Comment:
this is fine to delete since the website uses its own code it was not using this